plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakePageSource.java

```diff
@@ -61,23 +67,28 @@
     private final Block pathBlock;
     private final Block partitionsBlock;
     private final ConnectorPageSource delegate;
+    private List<DeltaLakeColumnHandle> notSynthesizedColumn;
+    private Set<Integer> missingIndexes;
+    private boolean isCDFEnabled;
 
     public DeltaLakePageSource(
-            List<DeltaLakeColumnHandle> columns,
+            List<DeltaLakeColumnHandle> relevantColumns,
+            List<DeltaLakeColumnHandle> allColumns,
```

Comment:
`allColumns` name seems improper because in case of not dealing with CDF, the `relevantColumns` will be actually the same as `columnsToRead`.

See `io.trino.plugin.deltalake.DeltaLakePageSourceProvider#createPageSource`